mocks/transaction.js

```diff
@@ -0,0 +1,43 @@
+const mockGetTransaction = jest.fn();
+const mockSetTransaction = jest.fn();
+const mockUpdateTransaction = jest.fn();
+const mockDeleteTransaction = jest.fn();
+
+class Transaction {
```

Comment:
Should this class also have `getAll`?
https://googleapis.dev/nodejs/firestore/latest/Transaction.html#getAll

Reply:
It definitely should! I had overlooked that.

The logic for that might be complicated. Could we perhaps add it in another PR after this series is through?

Reply:
It's probably just a loop over each provided ref. It shouldn't be too bad to throw that in this branch. Gimme a sec...

Reply:
Added in the latest commit. What do you think?